# Copyright (C) 2023 LEIDOS.
#
# Licensed under the Apache License, Version 2.0 (the "License"); you may not
# use this file except in compliance with the License. You may obtain a copy of
# the License at
#
# http://www.apache.org/licenses/LICENSE-2.0
#
# Unless required by applicable law or agreed to in writing, software
# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
# License for the specific language governing permissions and limitations under
# the License.

# Hz: The rate at which the LightBarManager Node will process message
spin_rate_hz: 10.0

# Operation mode: if false, other plugins are able to take control of the lightbar status
normal_operation: true

# Indicator IDs
# GREEN_SOLID
# GREEN_FLASH
# YELLOW_SIDES
# YELLOW_DIM
# YELLOW_FLASH
# YELLOW_ARROW_LEFT
# YELLOW_ARROW_RIGHT
# YELLOW_ARROW_OUT

# CDA Msg Type Ids
# TypeA
# TypeB
# TypeC
# TypeD

# Priority in descending order. Higher priority takes over lightbar
lightbar_priorities: ['lightbar_manager']

# Keys for lightbar_cda_to_ind_table, 1-to-1 with lightbar_ind_table
lightbar_cda_table: ['TypeA', 'TypeB', 'TypeC', 'TypeD']

# Values for lightbar_cda_to_ind_table, 1-to-1 with lightbar_cda_table
lightbar_ind_table: ['YELLOW_DIM','YELLOW_DIM','YELLOW_FLASH','YELLOW_SIDES']



    

